lib/src/core/util/card_constants.dart

```diff
@@ -0,0 +1,76 @@
+const String materialAppTitle = 'Hearthstone cards';
```

Comment:
This constants file could be divided in different files, one relative to dimens (Like fontSize, height, Colors), the second one relative to String constants, another relative to api service calls (apiBaseUrl, api endpoints ..), and other only to assets.

Reply:
Done.